x/gov/bucket.go

```diff
@@ -50,7 +52,7 @@ type ElectionRulesBucket struct {
 func NewElectionRulesBucket() *ElectionRulesBucket {
 	b := migration.NewBucket(packageName, "electnrule", orm.NewSimpleObj(nil, &ElectionRule{}))
 	return &ElectionRulesBucket{
-		VersioningBucket: orm.WithVersioning(orm.WithSeqIDGenerator(b, "id")),
+		VersioningBucket: orm.WithVersioning(orm.WithSeqIDGenerator(b, electionRuleSequence)),
```

Comment:
pp: as I wrote in my other comment, better drop the sequence here than being inconsistent in the api

Reply:
Here's the motivation behind this - if we ever want to use this bucket for anything else - we'll be forced to use manual id generation.

Since current implementations both use the same sequence - I'd say it is not such a bad thing to support both, but I do understand the idea.

WDYT?

There might be a better solution to this problem too, but I don't want it to drag along.